src/components/HomeLandingPage/HomeLandingPage.tsx

```diff
@@ -28,6 +30,7 @@ const HomeLandingPage = () => {
   // Fetch events from cache
   const { data } = useQuery(GET_ALL_EVENTS);
   const events: Array<EventType> = data ? data.events : [];
+  console.log(events);
```

Comment:
can you check if the linter catches this? i thought i had this rule on

Reply:
my bad, I forgot to run the linter